tools/pipelines/templates/build-npm-package.yml

```diff
@@ -261,20 +268,37 @@ stages:
               targetType: 'inline'
               workingDirectory: ${{ parameters.buildDirectory }}
               script: |
-                mkdir $(Build.ArtifactStagingDirectory)/pack/
+                mkdir $(Build.ArtifactStagingDirectory)/pack-scoped/
+                mkdir $(Build.ArtifactStagingDirectory)/pack-non-scoped/
                 if [ -f "lerna.json" ]; then
                   npx lerna exec --no-private --no-sort -- npm pack --unsafe-perm
-                  npx lerna exec --no-private --no-sort --parallel -- mv -t $(Build.ArtifactStagingDirectory)/pack/ ./*.tgz
+                  npx lerna exec --no-private --no-sort --parallel -- mv -t $(Build.ArtifactStagingDirectory)/pack-scoped/ ./*.tgz
```

Comment:
will this pack and copy all unscoped packages as well?

Reply:
The next step moves the non-scoped packages out to their own `pack-non-scoped` directory. This allows all of the packages to be moved in parallel, and then we just iterate through the ones that are non-scoped and move those separately

Reply:
i'd guess not, but is it a concern that new unscoped packages might get added and fall into here?  don't recall how much fluid-build guards against that

Reply:
The artifact would include them at that point, but then the publish itself would fail which I was thinking would be an indicator that the new package needs to be added to the parameters for it to be published